src/bin/gpfdist/gpfdist.c

```diff
@@ -1296,7 +1296,9 @@ session_get_block(const request_t* r, block_t* retblock, char* line_delim_str, i
 {
 	int 		size;
 	const int 	whole_rows = 1; /* gpfdist must not read data with partial rows */
-	struct fstream_filename_and_offset fos = {0};
+	struct fstream_filename_and_offset fos;
```

Comment:
Any strong opinion to use C99 explicit initializers?

```suggestion
	struct fstream_filename_and_offset fos = {.fname={0}, .line_number = 0, .foff=0};
```

I personally find `memset` a little obfuscating, _especially_ for stack-allocated objects, because they hide the individual field-initializations (alas, it's inevitable)

Reply:
I like the suggested change by @d

Reply:
Caveat: this does go (slightly) over the 80-line column width, so it might need to wrap at the assignment operator (or wherever `pgident` feels like it)

Reply:
Or just remove the initialization altogether? It's always set before being read and we typically never initialize such variables in postgres/gpdb.